src/mixins/child-apps.js

```diff
@@ -15,6 +15,14 @@ const ClassOptions = [
 export default {
 
   /**
+   * Set to true if a parent `App` should be able to restart this `App`.
+   *
+   * @type {Boolean|Function}
+   * @default false
+   */
+  restartWithParent: false,
```

Comment:
Actually I don't see how this is supporting "Can be added as an option when instantiated or defined on the `App` definition."  and I can't figure out why this option is defined here.  Adding `restartWithParent` should look a modify just about all of the places where `startWithParent` would be.

Reply:
Gotcha, I wasn't 100% sure that this documentation should go here, since higher up in the file it redirects to the app's lifecycle events. I'll move chat and change the wording.

Reply:
just clarfying `restartWithParent: false` doesn't belong in this file at all.

Reply:
also the documentation was correct...  it just isn't working as it was documented

Reply:
Ah, my apologies. I didn't fully understand your statement, so thanks for the clarification. I plan to move it into the app.js file, alongside the `startWithPatient` options.